components/proxy/src/test/kotlin/com/hotels/styx/services/YamlFileConfigurationServiceTest.kt

```diff
@@ -65,7 +66,7 @@ class YamlFileConfigurationServiceTest : FunSpec() {
         try {
             action(service)
         } finally {
-            service.stop().join()
+            service.stop().get(2, SECONDS)
```

Comment:
This could throw several exceptions here... makes me think we might want to implement Autocloseable.

Reply:
You mean, for the `StyxService`?